lua/script.lua

```diff
@@ -21,6 +21,8 @@ Script.clear = function()
     g.key = norns.none
   end
   g = nil
```

Comment:
also a question: why is g getting clobbered here?  one side-effect is that it's not available in the next `init()`.